constants/mediaquery.ts

```diff
@@ -0,0 +1 @@
+export const MOBILE_MEDIA = "@media screen and (max-width: 414px)";
```

Comment:
@axross It's good! I think I will use this after merging this request.

Reply:
Thanks! Go ahead!